### PR TITLE
testing build speed up

### PR DESCRIPTION
**Pull request by ejj**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,6 @@
 From quilt/ovs
 Maintainer Ethan J. Jackson
 
-RUN apt-get update \
-&& apt-get install -y --no-install-recommends iproute2 iptables \
-&& rm -rf /var/lib/apt/lists/*
-
```
Comment (kayousterhout) [inline Dockerfile]: Why does moving this speed up the test build?

File: .travis.yml [hunk 2]
```diff
@@ -12,11 +12,16 @@ install:
     - make go-get
 
 script:
-    - make all check lint format-check coverage docker-build-quilt docker-build-tester
+    - cat /proc/cpuinfo
```
Comment (kayousterhout) [inline .travis.yml]: Did you intend to leave this here or was it for temp debugging?

**Comment by ejj** [comment]
aaah, sorry @kayousterhout .. this isn't meant to be reviewed.  I'm using it to test out whether or not it speeds up the build in the travis build system.  I'll submit a formal PR later with the stuff that actually worked.

**Comment by kayousterhout** [comment]
Ah cool that makes more sense

**Pull request closed without merging**